sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -67,6 +68,9 @@ class QueueClient(StorageAccountHostsMixin, StorageEncryptionMixin):
     :keyword message_decode_policy: The decoding policy to use on incoming messages.
         Default value is not to decode messages. Other options include :class:`TextBase64DecodePolicy`,
         :class:`BinaryBase64DecodePolicy` or `None`.
+    :keyword str audience: The Storage account name for the audience to use when requesting tokens
+        for Azure Active Directory authentication. Only has an effect when credential is of type TokenCredential.
+        If not specified, the default audience "https://storage.azure.com/" will be used.
```

Comment:
Right now I'm leaning towards something like this still (Remove spaces around `<>` GitHub was weird and I had to add them). But not fully happy with it.

>The audience to use when requesting tokens for Azure Active Directory authentication. Only has an effect when credential is of type TokenCredential. Specify your Storage Account name to use the
https://< account >.blob.core.windows.net audience. Otherwise, if not specified, the default audience of https://storage.azure.com/ will be used.

Reply:
Will leave this open until we hit a happy docstring 😄